.pre-commit-config.yaml

```diff
@@ -16,9 +16,9 @@ repos:
         entry: ./tools/linter_go_routines.sh
         language: system
         types: [ go ]
-      - id: swagger
-        name: swagger
-        entry: ./tools/gen-code-from-swagger.sh
+      - id: golangci-lint
+        name: golangci-lint
+        entry: ./tools/golangci-lint.sh
```

Comment:
suggestion: can you need to add executable rights (`chmod +x`) to this file, and push it?

Reply:
I have just updated that, I ran into the same issue :-)